cmd/minikube/cmd/config/config.go

```diff
@@ -219,6 +219,21 @@ func configurableFields() string {
 	return strings.Join(fields, "\n")
 }
 
+// ListConfigMap list entries from config file
+func ListConfigMap(name string) (map[string]interface{}, error) {
+	configFile, err := config.ReadConfig()
+	if err != nil {
+		return nil, err
+	}
+	newImages := make(map[string]interface{})
```

Comment:
Would a "slice" of images work instead of a map? It looks like you only set "nil" as the value.

Reply:
No problem, I'll do it!